Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:focal
```

Comment:
we don't fully support ubuntu 22.04 right now, it still has unresolved issues with e.g. phabricator and newer versions of php.